readthedocs/doc_builder/python_environments.py

```diff
@@ -270,8 +271,8 @@ def install_core_requirements(self):
         )
 
     def install_user_requirements(self):
-        requirements_file_path = self.config.requirements_file
-        if not requirements_file_path:
+        requirements_file_path = self.config.python.requirements
+        if not requirements_file_path and requirements_file_path != '':
```

Comment:
This is needed for the new behavior of the requirements file, anyway, this doesn't affect the current behavior of v1 (v1 config returns null here).

Reply:
I want to echo what I said regarding using explicit `ignore` for this.

Also, `self.config.python.requirements` could return `IGNORE` when the YAML contains `''` (in case we won't change that for the explicit way).